pennylane/ops/cv.py

```diff
@@ -796,33 +807,257 @@ class CatState(CVOperation):
     """
     num_wires = 1
     num_params = 3
-    par_domain = 'R'
-    grad_method = 'F'
-
-
-all_ops = [
-    Beamsplitter,
-    ControlledAddition,
-    ControlledPhase,
-    Displacement,
-    Kerr,
-    CrossKerr,
-    QuadraticPhase,
-    Rotation,
-    Squeezing,
-    TwoModeSqueezing,
-    CubicPhase,
-    Interferometer,
-    CatState,
-    CoherentState,
-    FockDensityMatrix,
-    DisplacedSqueezedState,
-    FockState,
-    FockStateVector,
-    SqueezedState,
-    ThermalState,
-    GaussianState
-]
+    par_domain = "R"
+    grad_method = "F"
+
+
+# =============================================================================
+# Observables
+# =============================================================================
+
+
+class MeanPhoton(CVObservable):
+    r"""pennylane.expval.MeanPhoton(wires)
+    Returns the photon number expectation value.
+
+    This expectation command returns the value
+    :math:`\braket{\hat{n}}` where the number operator is
+    :math:`\hat{n} = \a^\dagger \a = \frac{1}{2\hbar}(\x^2 +\p^2) -\I/2`.
+
+    **Details:**
+
+    * Number of wires: 1
+    * Number of parameters: 0
+    * Observable order: 2nd order in the quadrature operators.
+    * Heisenberg representation:
+
+      .. math:: M = \frac{1}{2\hbar}\begin{bmatrix}
+            -\hbar & 0 & 0\\
+            0 & 1 & 0\\
+            0 & 0 & 1
+        \end{bmatrix}
+
+    Args:
+        wires (Sequence[int] or int): the wire the operation acts on.
+    """
+    num_wires = 1
+    num_params = 0
+    par_domain = None
+
+    ev_order = 2
+
+    @staticmethod
+    def _heisenberg_rep(p):
+        hbar = 2
+        return np.diag([-0.5, 0.5 / hbar, 0.5 / hbar])
+
+
+class X(CVObservable):
+    r"""pennylane.expval.X(wires)
+    Returns the position expectation value in phase space.
+
+    This expectation command returns the value :math:`\braket{\x}`.
```

Comment:
docstring will have to be updated to reflect refactor

Reply:
Fixed